src/Servers/IIS/IIS/test/IIS.FunctionalTests/ShadowCopyTests.cs

```diff
@@ -172,6 +172,91 @@ public async Task ShadowCopyE2EWorksWithFolderPresent()
             Assert.True(response.IsSuccessStatusCode);
         }
 
+        [ConditionalFact]
+        public async Task ShadowCopyE2EWorksWithOldFoldersPresent()
+        {
+            using var directory = TempDirectory.Create();
+            var deploymentParameters = Fixture.GetBaseDeploymentParameters();
+            deploymentParameters.HandlerSettings["experimentalEnableShadowCopy"] = "true";
+            deploymentParameters.HandlerSettings["shadowCopyDirectory"] = directory.DirectoryPath;
+            var deploymentResult = await DeployAsync(deploymentParameters);
+
+            // Start with 1 to exercise the incremental logic
+            DirectoryCopy(deploymentResult.ContentRoot, Path.Combine(directory.DirectoryPath, "1"), copySubDirs: true);
+
+            var response = await deploymentResult.HttpClient.GetAsync("Wow!");
+            Assert.True(response.IsSuccessStatusCode);
+
+            using var secondTempDir = TempDirectory.Create();
+
+            // copy back and forth to cause file change notifications.
+            DirectoryCopy(deploymentResult.ContentRoot, secondTempDir.DirectoryPath, copySubDirs: true);
+            DirectoryCopy(secondTempDir.DirectoryPath, deploymentResult.ContentRoot, copySubDirs: true);
+
+            response = await deploymentResult.HttpClient.GetAsync("Wow!");
```

Comment:
Why even try a request here?

Reply:
Mostly wanted to see how graceful/what would happen, I was hoping it would consistently return Application Shutting Down like it does locally.  I could remove it, but we get some coverage that we get a 503 or 200 at least here, so I felt it has some value